### PR TITLE
feat: import deployment plan from a file ; rebase done

### LINKED ISSUE #397: feat: Exporting and/or importing a plan from a file
Hi,

It would be nice to import a plan from a file with a command like ```tdp plan import [file]```.

Maybe the export command would be useful, at least to obtain logs of a given deployment.

### PR DESCRIPTION
**Pull request by mdrutel**
#### Which issue(s) this PR fixes

Fixes #397 

#### Agreements

- [X] I hereby declare this contribution to be licensed under the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0).
- [X] I hereby agree to grant [TOSIT](https://www.tosit.io/) a copyright license to use my contributions.

### CODE REVIEW

File: tdp/cli/commands/plan/import_file.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+from tdp.cli.queries import get_planned_deployment
+from tdp.cli.session import get_session
+from tdp.cli.utils import collections, database_dsn, parse_file
+from tdp.core.models.deployment_model import DeploymentModel
+
+if TYPE_CHECKING:
+    from tdp.core.collections import Collections
+
+logger = logging.getLogger("tdp").getChild("edit")
```
Comment (PaulFarault) [inline tdp/cli/commands/plan/import_file.py:19]: You should use:

```suggestion
logger = logging.getLogger(__name__)
```

Thought I had fixed it everywhere but it seems that I have forgot a few places.

File: tdp/cli/commands/plan/import_file.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+                    if planned_deployment:
+                        deployment.id = planned_deployment.id
+                    session.merge(deployment)
+                    session.commit()
+                    click.echo("Deployment plan successfully imported.")
+                except Exception as e:
+                    logger.error(str(e))
+        except Exception as e:
+            logger.error(str(e))
```
Comment (PaulFarault) [inline tdp/cli/commands/plan/import_file.py:56]: Those are not required since #482. You can still use `try`/`except` but to catch specific exception.

**Review by PaulFarault — COMMENTED** [review]
Well done!

**Review by PaulFarault — CHANGES_REQUESTED** [review]
Well done!

File: tdp/cli/commands/plan/import_file.py (hunk 1 above)
Comment (mdrutel) [reply tdp/cli/commands/plan/import_file.py]: done ; I also fixed the other file edit.py

**Comment by mdrutel** [comment]
I also added 1 comment, and fix the test :)

**Review by PaulFarault — APPROVED** [review]

**Review by SteBaum — APPROVED** [review]
I have looked at the code and also tested it and it works for me.

**Review by rpignolet — APPROVED** [review]

**Pull request merged**